backend/game/cards/discard_cards/victory_point_manipulation_cards/test_fire_blast.py

```diff
@@ -0,0 +1,13 @@
+from game.cards.discard_cards.victory_point_manipulation_cards.fire_blast import FireBlast
+
+
+def test_fire_blast_subtracts_2_health(player, five_players):
+    for other_players in five_players:
+        other_players.update_health_by(12)
```

Comment:
Why did we have to add 12 to all of the players? Why not just assert that their end health is starting health - 2?

Reply:
change made